storage/storage.go

```diff
@@ -28,3 +29,14 @@ import (
 type APICallWriter interface {
 	WriteAPICall(ctx context.Context, l *ctlog.Log, apiCall *apicall.APICall) error
 }
+
+// STHWriter is an interface for storing STHs received from a CT Log.
+type STHWriter interface {
+	WriteSTH(ctx context.Context, l *ctlog.Log, sth *ct.SignedTreeHead, errs []error) error
+}
+
+// APICallSTHWriter represents a type that can store API Calls and store STHs.
+type APICallSTHWriter interface {
```

Comment:
This interface might be better living with `sthgetter`, since it basically defines the interfaces that it uses.

Reply:
Hmm sure.  Do you think keeping the base building block interfaces in here is fine, but then the combined ones could go in the modules that use them? (assuming those particular combinations are only used by one module - although I guess we can always move them back if multiple modules need the same things...)

Reply:
Yes that's what I had in mind.

Reply:
Wonderful :)  Done.